### DISCUSSION TITLE
Migrating `buildSrc` dependencies to "Version Catalog"

### DISCUSSION THREAD

**Discussion by mohsenoid** [GetStream/stream-chat-android#5274] [Ideas]
While reading the source code and checking the dependencies I noticed that you haven't migrated to the "Version Catalog" yet. I wanted to suggest replacing the `buildSrc` dependencies with Version Catalog, as it's well-supported by the IDE.

**Comment by JcMinarro** [comment]
Hello @mohsenoid 
Yes, it is in our roadmap. We already migrate other projects as [VideoSDK](https://github.com/GetStream/stream-video-android) or [Stream Result](https://github.com/GetStream/stream-result), but this project wasn't yet.
PR are welcome :upside_down_face:
